packages/agent/tests/dwn-manager.spec.ts

```diff
@@ -530,7 +532,9 @@ describe('DwnManager', () => {
           target         : identity.did,
           messageType    : 'RecordsRead',
           messageOptions : {
-            recordId: message.recordId
+            filter: {
+              recordId: message.recordId
```

Comment:
I _think_ this test is failing because it's using a version of `dwn-server` that isn't updated to the latest version of `dwn-sdk-js`